storage/innobase/xtrabackup/src/xbcloud/xbcloud-t.cc

```diff
@@ -77,7 +78,8 @@ TEST(s3_client, basicDNSv4) {
 
 TEST(s3_client, basicPATHv4) {
```

Comment:
Should we add similar test for azure APIs?

Reply:
added